### ISSUE TITLE
R-Wert not in line with guthub repo nowcast

### ISSUE THREAD

**Issue by btreut** [corona-warn-app/cwa-documentation#872]

**Comment by MikeMcC399** [comment]
@btreut 

After researching the data on https://github.com/robert-koch-institut/SARS-CoV-2-Nowcasting_und_-R-Schaetzung/blob/main/Nowcast_R_aktuell.csv it is clear the the differences in trends which we see in the CWA app compared to looking at a single file on GitHub are due to recalculation.

For the last 5 weekdays the app displayed the following values taken from the daily reports on https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Situationsberichte/Gesamt.html?nn=13490888

RKI Daily report   date | 7-Day R-value | Cases up to
-- | -- | --
14.03.2022 | 1,00 | 10.03.2022
15.03.2022 | 0,99 | 11.03.2022
16.03.2022 | 0,98 | 12.03.2022
17.03.2022 | 0,99 | 13.03.2022
18.03.2022 | 1,00 | 14.03.2022

You can find exactly these values in https://github.com/robert-koch-institut/SARS-CoV-2-Nowcasting_und_-R-Schaetzung/blob/main/Nowcast_R_aktuell.csv however, you need to take the data from each daily archive file, not from the latest file. CWA displays the best information it has each day. It does not replace the historical data for previous days when it is calculating trends though.

GitHub file date | PS_7_Tage_R_Wert | for day
-- | -- | --
14.03.2022 | 1,00 | 09.03.2022
15.03.2022 | 0,99 | 10.03.2022
16.03.2022 | 0,98 | 11.03.2022
17.03.2022 | 0,99 | 12.03.2022
18.03.2022 | 1,00 | 13.03.2022

The recalculated previous days in the file for 18.03.2022 have different values, and therefore the trend is not the same as the trend shown in the app.

GitHub file date | PS_7_Tage_R_Wert | for day
-- | -- | --
18.03.2022 | 1,10 | 09.03.2022
18.03.2022 | 1,09 | 10.03.2022
18.03.2022 | 1,06 | 11.03.2022
18.03.2022 | 1,03 | 12.03.2022
18.03.2022 | 1,00 | 13.03.2022

---

In summary, the 7-Day R Value data and trends shown in the Corona-Warn-App are correct. In order to reconcile the data displayed in the app with the data on GitHub it is necessary to use multiple `.csv` files from the [Archiv](https://github.com/robert-koch-institut/SARS-CoV-2-Nowcasting_und_-R-Schaetzung/tree/main/Archiv) folder.

Thank you for raising this issue. I hope that your question is now finally answered!

**Comment by MikeMcC399** [comment]
It may be worth requesting RKI to provide a single table showing all the historical values of the "7-Day R Value" which have been published to the daily situation report and shown in the Corona-Warn-App. This could be added as a new column to [Nowcast_R_aktuell.csv](https://github.com/robert-koch-institut/SARS-CoV-2-Nowcasting_und_-R-Schaetzung/blob/main/Nowcast_R_aktuell.csv) on GitHub and it would save having to access multiple files in order to build this data-set. If somebody wants to make that request it should probably go through [SARS-CoV-2-Nowcasting_und_-R-Schaetzung/issues](https://github.com/robert-koch-institut/SARS-CoV-2-Nowcasting_und_-R-Schaetzung/issues) or the contacts listed in [SARS-CoV-2-Nowcasting und -R-Schätzung](https://github.com/robert-koch-institut/SARS-CoV-2-Nowcasting_und_-R-Schaetzung#sars-cov-2-nowcasting-und--r-sch%C3%A4tzung).

Without such a table it will always be necessary to access multiple archived files to retrieve the historical values which have been shown daily in the statistics tiles of the Corona-Warn-App.

I'm not planning to make this request personally as I do not have this need.

**Comment by btreut** [comment]
@MikeMcC399 thanks for following up the topic. Yes, I know that the values change in time, but these also get more precise and the confidence range narrows. I know that I have to look at the archived older versions for getting the published number. I had previously a similar discussion with the guys at RKI (quite some time ago) about that topic and after that the values displayed in CWA were some time in line with the ones displayed in the nowcasting github repo. And yes, I felt that I am not taken seriously. When I started this issue, I had the  impression that the numbers were again diverging.
